src/backend/joanie/core/admin.py

```diff
@@ -83,4 +83,13 @@ class EnrollmentAdmin(admin.ModelAdmin):
 class AddressAdmin(admin.ModelAdmin):
     """Admin class for the Address model"""
 
-    list_display = ("name", "address", "postcode", "city", "country", "owner", "main")
+    list_display = (
+        "name",
```

Comment:
should we rename this to title for clarity?

Reply:
Or I rename `fullname` `recipient`?

Reply:
I see what you mean... it's not the fullname of the address either. But the placeholder above the field will show "full name" in the form so it doesn't disturb me. Since we use "title" for course runs, and I like title for a field that is used as a personal identifier of the object :man_shrugging:

Reply:
Finally I renamed address `name` field into `title`.